string uuid
uint32 texture_index
---
sensor_msgs/Image texture
